messenger/http_transporter_test.go

```diff
@@ -161,23 +151,20 @@ func TestTransporterStartAndRcvd(t *testing.T) {
 		}
 	}()
 
-	errch := receiver.Start()
+	rcvPid, errch := receiver.Start()
 	defer receiver.Stop(false)
 	assert.NotNil(t, errch)
 
 	time.Sleep(time.Millisecond * 7) // time to catchup
 
 	// setup sender (client) process
-	sndUpid, err := upid.Parse(fmt.Sprintf("mesos1@localhost:%d", getNewPort()))
-	assert.NoError(t, err)
-
-	sender := NewHTTPTransporter(sndUpid, nil)
+	sender := NewHTTPTransporter(upid.UPID{ID: "mesos1", Host: "localhost"}, nil)
 	msg := &Message{
-		UPID:         rcvPid,
+		UPID:         &rcvPid,
```

Comment:
Probably not, but that can be a separate refactoring.

On Thu, Sep 24, 2015 at 5:44 AM, Tomás Senart <notifications@github.com>
wrote:

> In messenger/http_transporter_test.go
> <https://github.com/mesos/mesos-go/pull/189#discussion_r40301640>:
>
> >  	msg := &Message{
> > -		UPID:         rcvPid,
> > +		UPID:         &rcvPid,
>
> Should we still use UPID pointers in Message structs?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/mesos/mesos-go/pull/189/files#r40301640>.
>